src/core-layers/screen-grid-layer/screen-grid-layer.js

```diff
@@ -20,16 +20,20 @@
 
 import {Layer, experimental} from '../../core';
 const {defaultColorRange, quantizeScale} = experimental;
-
+const {ScreenGridAggregator} = experimental;
 import {GL, Model, Geometry} from 'luma.gl';
 import {lerp} from './../../core/utils/math-utils';
 import log from './../../core/utils/log';
 
-import vs from './screen-grid-layer-vertex.glsl';
+import vsCpu from './screen-grid-layer-vertex.glsl';
+import vsGpu from './screen-grid-layer-vertex-gpu.glsl';
 import fs from './screen-grid-layer-fragment.glsl';
 
+const LOG_LEVEL = 0;
+
 const defaultProps = {
   cellSizePixels: 100,
+  gpuAggregation: false,
```

Comment:
would _useGPU_ or _useGPUAggregation_ read better here?
useGPU passed in props, and after internal checking in line 133, usingGPU(Aggregation) as internal state.